.changeset/clean-seals-sip.md

```diff
@@ -0,0 +1,5 @@
+---
+'@roadiehq/scaffolder-backend-module-utils': patch
```

Comment:
I believe this is actually a major change even though it's correcting the types? (feel free to disagree) e.g. what currently compiles (but doesn't work as expected) will not now compile? I understand the PR which switched the library was also a patch but that seems to have been a mistake

Reply:
Agreed, I think the original PR should have been a breaking change, I'm happy to update accordingly.

Can I just clarify this statement in the contributing doc:

> Select impact of change that you're introducing, using minor for breaking changes and patch otherwise.

Is this still the case, all breaking changes are *minor* version bumps? Any reason not to use semantic versioning?

Reply:
There's a convention (not sure how widely followed, it was new to me) that pre version 1 (e.g. 0.2.3) that minors can be used for breaking changes as you'd likely want to make a bunch of breaking changes prior to a formal release. 

I just checked Backstage and they've qualified this line:
```
Select impact of the change you're introducing. If the package you are changing is at version 0.x, use minor for breaking changes and patch otherwise. If the package is at 1.0.0 or higher, use major for breaking changes, minor for backwards compatible API changes, and patch otherwise. See the [Semantic Versioning specification](https://semver.org/#semantic-versioning-specification-semver) for more details.
```

Frankly, I'd be inclined to always follow semver as users are likely going to pull in v0 packages anyway. I'll discuss internally we my team and we'll update this doc.

Reply:
Done, both `major` changeset and update to CONTRIBUTING.md